docs/starters.yml

```diff
@@ -98,6 +106,13 @@
   features:
     - Comes with React Helmet for adding site meta tags
     - Includes plugins for offline support out of the box
+- url: https://github.com/gatsbyjs/gatsby-starter-theme-workspace
+  repo: https://github.com/gatsbyjs/gatsby-starter-theme-workspace
+  description: official workspace theme
+  tags:
+    - Official
+  features:
+    - Minimalist plugin to make content stand out
```

Comment:
This isn't entirely accurate. The Theme Workspace is a starter to help you get started building a Gatsby Theme. 

This could possibly say something like:
- preconfigured theme and site to get started developing a theme
- yarn workspaces set up to connect the example site to the theme